src/Mod/Path/PathTests/TestPathPost.py

```diff
@@ -152,3 +154,317 @@ def testSplitArcs(self):
         self.assertTrue(
             len([c for c in results.Commands if c.Name in ["G2", "G3"]]) == 0
         )
+
+
+def dumpgroup(group):
+    print("====Dump Group======")
+    for i in group:
+        print(i[0])
+        for j in i[1]:
+            print(f"--->{j.Name}")
+    print("====================")
+
+
+class TestBuildPostList(unittest.TestCase):
+    """
+    The postlist is the list of postprocessable elements from the job.
+    The list varies depending on
+        -The operations
+        -The tool controllers
+        -The work coordinate systems (WCS) or 'fixtures'
+        -How the job is ordering the output (WCS, tool, operation)
+        -Whether or not the output is being split to multiple files
+    This test case ensures that the correct sequence of postable objects is
+    created.
+
+    The list will be comprised of a list of tuples. Each tuple consists of
+    (subobject string, [list of objects])
+    The subobject string can be used in output name generation if splitting output
+    the list of objects is all postable elements to be written to that file
+
+    """
+
+    testfile = FreeCAD.getHomePath() + "Mod/Path/PathTests/test_filenaming.fcstd"
+    doc = FreeCAD.open(testfile)
+    job = doc.getObjectsByLabel("MainJob")[0]
+
+    def test000(self):
+
+        # check that the test file is structured correctly
+        self.assertTrue(len(self.job.Tools.Group) == 2)
+        self.assertTrue(len(self.job.Fixtures) == 2)
+        self.assertTrue(len(self.job.Operations.Group) == 3)
+
+        self.job.SplitOutput = False
+        self.job.OrderOutputBy = "Operation"
+
+    def test010(self):
+        # check that function returns correct hash
```

Comment:
Does this comment reflect the functionality of the test? Seems like the test validate the structure of what buildPostList returns.

Reply:
Will change.